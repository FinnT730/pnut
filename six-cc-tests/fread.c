int main() {
  int MAX_SIZE = 1000;
  int_ptr f;
  char_ptr s;
  int len;
  int h;
  s = malloc(MAX_SIZE);
  f = fopen("six-cc-tests/fread.c", 0);
  len = fread(s, 1, MAX_SIZE - 1, f);
  s[len] = 0;
  fclose(f);
  printf("===== Read content =====\n%s\n", s);
}
