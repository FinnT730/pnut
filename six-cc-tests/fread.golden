===== Read content =====
int main() {
  int MAX_SIZE = 200;
  int_ptr f;
  char_ptr s;
  int len;
  int h;
  s = malloc(MAX_SIZE);
  f = fopen("six-cc-tests/fread.c", 0);
  len = fread(f, s, MAX_SIZE - 1);
  s[len] = 0;
  fc
